ios/ble/Utility/chunker.swift

```diff
@@ -89,9 +92,11 @@ class Chunker {
      |                       |                             |                                                                         |
      +-----------------------+-----------------------------+-------------------------------------------------------------------------+
      */
+     
+    private func frameChunk(seqIndex: Int, chunkLength: Int, fromIndex: Int, toIndex: Int) -> Data {
+        let seqNumber = seqIndex + 1
+        print("fetching chunk size:",toIndex,"-", fromIndex,"}, chunkSequenceNumber(1-indexed):", seqNumber)
```

Comment:
Remove the unwanted print statements

Reply:
i discussed with shree thaanu and removed the unwanted print statements. There is only 1 print statement in this file which is needed.